### DISCUSSION TITLE
Multiple = true

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#32630] [Q&A]
Is there a difference between `multiple=true` and `multiple=multiple`

I can see both being used

**Comment by joomdonation** [comment]
From what I can see from the code, there is no difference between the two. For reference, code which handles it is:

https://github.com/joomla/joomla-cms/blob/4.0-dev/libraries/src/Form/FormField.php#L534-L545
https://github.com/joomla/joomla-cms/blob/4.0-dev/libraries/src/Form/Rule/EmailRule.php#L75

So I would say that it is safe to change to the one you choose to be consistent.

**Reply by dgrammatiko** [reply]
@brianteeman multiple as an HTMLElement is a boolean: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/multiple So maybe it would be better to keep the `multiple=true`

**Reply by brianteeman** [reply]
The main use of multiple=multiple is with com_workflows and its plugins
